roles/edpm_ovn/templates/kolla_ovn_metadata_agent.yaml.j2

```diff
@@ -1,4 +1,4 @@
-command: "/usr/bin/networking-ovn-metadata-agent --config-file {{ edpm_ovn_metadata_agent_config_dir }}/neutron.conf --config-file {{ edpm_ovn_metadata_agent_config_dir }}/plugins/networking-ovn/networking-ovn-metadata-agent.ini --log-file={{ edpm_ovn_metadata_agent_log_dir }}/ovn-metadata-agent.log"
+command: "networking-ovn-metadata-agent"
```

Comment:
would be good to change the binary also to "neutron-ovn-metadata-agent" as it's provided by neutron sub rpms, may be as followup.